api_v3/lib/KalturaErrors.php

```diff
@@ -582,6 +582,12 @@ class KalturaErrors extends APIErrors
 	 * Live reports Service
 	 */
 	const LIVE_REPORTS_WS_FAILURE = 'LIVE_REPORTS_WS_FAILURE;;failed to retrieve live analytics';
+	
+	/*
+	 * Analytics Service
+	 */
+	const ANALYTICS_QUERY_FAILURE = 'REPORTS_QUERY_FAILURE;ERROR_MSG;failed to retrieve analytics data - [@ERROR_MSG@]';
```

Comment:
The attributes in the message created to enable future multilinguality, have attribute that contains description kind of missing the point.
Would it be possible to reflect all the different errors in the kAnalony as different errors in the API?